benchmarks/src/jmh/kotlin/benchmarks/flow/FlowFlattenMergeBenchmark.kt

```diff
@@ -0,0 +1,52 @@
+package benchmarks.flow
+
+import doWork
+import kotlinx.coroutines.flow.*
+import kotlinx.coroutines.runBlocking
+import org.openjdk.jmh.annotations.*
+import java.util.concurrent.TimeUnit
+
+@Warmup(iterations = 3)
+@Measurement(iterations = 10)
+@BenchmarkMode(Mode.Throughput)
```

Comment:
For microbenchmarks (and I assume this is one) please add `time = 1` and decrease amount of iterations. It doesn't make any sense to burn one's CPU for nothing when for such simple benchmark steady state is reached within a couple of iterations.

Also, it has a too broad set of default parameters, thus this benchmark takes way too much time to complete.
Default time unit is incorrect, the output of the benchmark is "10⁻⁴ ops/us" for all iterations